raiden-ts/src/path/epics.ts

```diff
@@ -318,3 +437,34 @@ export const pfsServiceRegistryMonitorEpic = (
       ),
     ),
   );
+
+export const udcBalanceEpic = (
+  action$: Observable<RaidenAction>,
+  state$: Observable<RaidenState>,
+  { userDepositContract }: RaidenEpicDeps,
+): Observable<ActionType<typeof udcBalanceUpdate | typeof udcBalanceFetchFailed>> => {
+  return action$.pipe(
+    filter(isActionOf(udcBalanceFetch)),
+    debounceTime(10e3),
+    withLatestFrom(state$),
+    mergeMap(([{}, state]) => {
+      return from(userDepositContract.balances(state.address)).pipe(
```

Comment:
Also, don't forget `functions`. I'm not sure ethers also expose as direct method, but even if it does, for consistency, it's better to call method from `contract.functions`